index.js

```diff
@@ -0,0 +1,239 @@
+  try {
+  if (msg.length < 4) return doHelp(tweet, msg);
+  let address = msg[2];
+  let amount = getValidatedAmount(msg[3]);
+  if (amount === null) {
+    return await T.post("statuses/update", {
+      status: `@${tweet.user.screen_name} I don´t know how to withdraw that many credits...`,
+      in_reply_to_status_id: tweet.id_str
+    });
+  }
+  let txId = await lbry.sendFrom(id(tweet.user.id_str), address, amount);
+  await T.post("statuses/update", {
+    status: `@${tweet.user.screen_name} You withdrew ${amount} LBC to ${address}. \n${txLink(txId)}`,
+    in_reply_to_status_id: tweet.id_str
+  });
+  logger.info(
+    `User ${
+      tweet.user.screen_name
+    } withdraw ${amount} LBC to ${address}, tweet id: ${tweet.id_str}`
+  );
+  } catch (e) {
+    logger.error(e);
+  }
+}
+async function doTip(tweet, msg) {
+  try {
+    if (msg.length < 3) {
+      return doHelp(tweet, msg);
+    }
+    const amount = getValidatedAmount(msg[3]);
+    if (amount === null) {
+      return await T.post("statuses/update", {
+        status: `@${tweet.user.screen_name} I don´t know how to tip that many credits...`,
+        in_reply_to_status_id: tweet.id_str
+      });
+    }
+    const userToTip = tweet.entities.user_mentions.find(u => `@${u.screen_name}` === msg[2]).id_str;
+    await getAddress(id(userToTip)) // Call this to ensure user has an account.
+    if (userToTip === null) {
+      return await T.post("statuses/update", {
+        status: `@${tweet.user.screen_name} I could not find that user...`,
+        in_reply_to_status_id: tweet.id_str
+      });
+    }
+    const balanceFromUser = await lbry.getBalance(id(tweet.user.id_str), config.get("bot.requiredConfirms"));
+    if (balanceFromUser < amount) {
+      return await T.post("statuses/update", {
+        status: `@${tweet.user.screen_name} You tried to tip, but you are missing ${amount-balanceFromUser} LBC.`,
+        in_reply_to_status_id: tweet.id_str
+      });
+    }
+    const txId = await lbry.move(
+      id(tweet.user.id_str),
+      id(userToTip),
+      Number(amount)
+    );
+    await T.post("statuses/update", {
+      status: `@${tweet.user.screen_name} Tipped ${
+        msg[2]
+        } ${amount} LBC! \n See https://lbry.io/faq/tipbot-twitter for more information.`,
+      in_reply_to_status_id: tweet.id_str
+    });
+    logger.info(
+      `@${tweet.user.screen_name}(${tweet.user.id_str}) tipped ${
+        msg[2]
+      }(${userToTip}) ${amount} LBC.`
+    );
+  } catch (e) {
+    logger.error(e);
+  }
+}
+
+async function getAddress(userId) {
+  try {
+    let uAddresses = await lbry.getAddressesByAccount(userId);
+    if (uAddresses.length > 0) return uAddresses[0];
+    let nAddress = await lbry.getNewAddress(userId);
+    return nAddress;
+  } catch (e) {
+    logger.error(e);
+  }
+}
+
+function getValidatedAmount(amount) {
+  amount = amount.trim();
+  if (amount.toLowerCase().endsWith("lbc")) {
+    amount = amount.substring(0, amount.length - 3);
```

Comment:
`amount = amint.split("lbc")[0];` probably works too without having to worry too much about spaces or anything after "lbc".

Though it would have to be tested

Reply:
Would probably work, want to keep it like this for now, as this function is the same in the discord tipbot :)